shared/templates/selinux_booleans.csv

```diff
@@ -74,13 +74,13 @@ glance_use_fusefs,
 global_ssp,
 gluster_anon_write,
 gluster_export_all_ro,
-gluster_export_all_rw,
+gluster_export_all_rw,disable
 gpg_web_anon_write,
-gssd_read_tmp,
-guest_exec_content,
+gssd_read_tmp,enable
+guest_exec_content,disable
 haproxy_connect_any,
 httpd_anon_write,
-httpd_builtin_scripting,
+httpd_builtin_scripting,disable
```

Comment:
Not here. A httpd specific STIG should turn it on as needed.

Admittedly, this is kind of a crappy way to 'force' App Server STIGs to manage their SELinux booleans when they create their STIGs.